src/webui/api/serialize/serialize_torrent.cpp

```diff
@@ -166,6 +166,7 @@ QVariantMap serialize(const BitTorrent::Torrent &torrent)
         {KEY_TORRENT_COMMENT, torrent.comment()},
         {KEY_TORRENT_PRIVATE, (torrent.hasMetadata() ? torrent.isPrivate() : QVariant())},
         {KEY_TORRENT_TOTAL_SIZE, torrent.totalSize()},
-        {KEY_TORRENT_HAS_METADATA, torrent.hasMetadata()}
+        {KEY_TORRENT_HAS_METADATA, torrent.hasMetadata()},
+        {KEY_TORRENT_PERCENT_SELECTED, torrent.hasMetadata() ? (torrent.wantedSize() * 100) / torrent.totalSize() : -1},
```

Comment:
```suggestion
        {KEY_TORRENT_PERCENT_SELECTED, (torrent.hasMetadata() ? (qreal(torrent.wantedSize()) / torrent.totalSize()) : -1)},
```

Reply:
@Chocobo1, don't forget that qBittorrent coding style uses encourage `static_cast<qreal>` in the similar cases.